chaos-experiments/docker-compose/broker-outage/experiment.json

```diff
@@ -0,0 +1,34 @@
+{
+  "version": "0.1.0",
+  "title": "Zeebe broker outage experiment",
+  "description": "Zeebe should be fault-tolerant when one broker dies",
+  "contributions": {
+    "reliability": "high",
+    "availability": "high"
+  },
+  "steady-state-hypothesis":
+  {
+    "title": "Zeebe is alive",
+    "probes": [
+      {
+        "name": "should-be-able-to-create-workflow-instances",
+        "type": "probe",
+        "tolerance": 0,
+        "provider": {
+          "type": "process",
+          "path": "../../scripts/docker-compose/verify-steady-state.sh",
+          "timeout": 60
+        }
+      }]
+  },
+  "method": [
+    {
+      "type": "action",
+      "name": "terminate-random-broker",
+      "provider": {
+        "type": "process",
+        "path": "../../scripts/docker-compose/kill-random-broker.sh"
+      }
+    }],
+  "rollbacks": []
```

Comment:
Since we expecting docker-compose, I assume this setup https://github.com/zeebe-io/zeebe-docker-compose/blob/master/cluster/docker-compose.yml ? We should bring the container back on rollback. In kubernetes it happens automatically if you delete a pod, which is the reason we don't need it there.

Reply:
Sure, let me check what docker-compose supports here or if I have to do it manually.

Reply:
Ok I've updated my PR and added a clean, self-contained, working chaostoolkit-test setup with docker-compose, rearranged files a bit.